src/main/java/seedu/address/model/AddressBook.java

```diff
@@ -85,6 +143,30 @@ public void setPerson(Person target, Person editedPerson) {
         persons.setPerson(target, editedPerson);
     }
 
+    /**
+     * Replaces the given person {@code target} in the list with {@code editedProject}.
+     * {@code target} must exist in the address book.
+     * The project identity of {@code editedProject} must not be the same as another
+     * existing project in the address book.
+     */
+    public void setProject(Project target, Project editedProject) {
+        requireNonNull(editedProject);
+
+        projects.setProject(target, editedProject);
+    }
+
+    /**
+     * Replaces the given person {@code target} in the list with {@code editedIssue}.
```

Comment:
should be given issue instead of given person?
```suggestion
     * Replaces the given issue {@code target} in the list with {@code editedIssue}.
```